### PR TITLE
Only expose v0.1.5 and v0.2.1 in 2.5.6

### PR DESCRIPTION
**Pull request by aiyengar2**

### CODE REVIEW

File: charts/rancher-monitoring/v0.1.4/questions.yml [hunk 1]
```diff
@@ -1 +1,2 @@
 rancher_min_version: 2.4.9-rc1
+rancher_max_version: 2.5.6-rc1
```
Comment (deniseschannon) [inline charts/rancher-monitoring/v0.1.4/questions.yml]: Please update 2.5.5

File: charts/rancher-monitoring/v0.2.0/questions.yml [hunk 2]
```diff
@@ -1 +1,2 @@
 rancher_min_version: 2.5.4-rc1
+rancher_max_version: 2.5.6-rc1
```
Comment (deniseschannon) [inline charts/rancher-monitoring/v0.2.0/questions.yml]: Please update 2.5.5

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: charts/rancher-monitoring/v0.1.4/questions.yml (hunk 1 above)
Comment (aiyengar2) [reply charts/rancher-monitoring/v0.1.4/questions.yml]: updated

File: charts/rancher-monitoring/v0.2.0/questions.yml (hunk 2 above)
Comment (aiyengar2) [reply charts/rancher-monitoring/v0.2.0/questions.yml]: updated

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**